data/Netherlands/House_of_Representatives/sources/instructions.json

```diff
@@ -75,6 +75,21 @@
         "from": "group-wikidata",
         "source": "manual/group_wikidata.csv"
       }
+    },
+    {
+      "file": "morph/twitter.csv",
+      "create": {
+        "from": "morph",
+        "scraper": "everypolitician-scrapers/twitter-list-netherlands",
+        "query": "SELECT * FROM data ORDER BY id"
+      },
+      "type": "person",
+      "source": "https://twitter.com/Simon_Tol/lists/kabinet-rutte-ii, https://twitter.com/PolitwoopsNL/lists/tweede-kamer-overig and https://twitter.com/PolitwoopsNL/lists/tweede-kamer-overig",
```

Comment:
I'm afraid this can't simply be a sentence like this: the entry has to be an actual URL. What you probably want to do here is cherry-pick https://github.com/everypolitician/everypolitician-data/pull/15353/commits/73b633e368a280834dcb4eb7ff381c513b06c6b6 and turn this into a list. 

(NB the 2nd and third entry here, and in the PR description, are the same URL…)